includes/admin/reports/class-donor-reports-table.php

```diff
@@ -184,9 +184,11 @@ public function column_default( $item, $column_name ) {
 				break;
 
 			case 'num_donations' :
-				$value = '<a href="' .
-				         admin_url( 'edit.php?post_type=give_forms&page=give-payment-history&user=' . urlencode( $item['email'] )
-				         ) . '">' . esc_html( $item['num_donations'] ) . '</a>';
+				$value = sprintf(
+					'<a href="%s">%s</a>',
```

Comment:
@DevinWalker It will be good if we show donor payment history by `id` instead of `email`